lib/avromatic.rb

```diff
@@ -27,9 +27,15 @@ def self.configure
 
   def self.build_schema_registry
     raise 'Avromatic must be configured with a registry_url' unless registry_url
-    AvroTurf::CachedConfluentSchemaRegistry.new(
-      AvroTurf::ConfluentSchemaRegistry.new(registry_url, logger: logger)
-    )
+    if use_cacheable_schema_registration
```

Comment:
I vote for `use_schema_fingerprint_lookup` in case other schema registries introduce other forms of caching.